src/dmd/root/ctfloat.d

```diff
@@ -216,17 +216,20 @@ extern (C++) struct CTFloat
     }
 
     // Constant real values 0, 1, -1 and 0.5.
-    __gshared real_t zero;
-    __gshared real_t one;
-    __gshared real_t minusone;
-    __gshared real_t half;
-
-    @trusted
-    static void initialize()
-    {
-        zero = real_t(0);
-        one = real_t(1);
-        minusone = real_t(-1);
-        half = real_t(0.5);
+    version (CRuntime_Microsoft)
```

Comment:
You should use `static if (real.sizeof <= 8)`, or maybe `static if (is(longdouble = real)`.